### DISCUSSION TITLE
Pen mode isssue (premium)

### DISCUSSION THREAD

**Discussion by phourglass** [AppGround-io/bluetooth-keyboard-and-mouse-support#173] [Bugs & Problems]
Hello,

I have a galaxy note 10 lite with a built in pen (s-pen) + the premium version in pen mode.

The mouse moves with the pen as intended, including when the pen is just above the screen (the location on the phone is visualized as a small circle).  
However, when I touch the screen with the pen, there is no click, which means no drawing. This makes the feature essentially unusable.

Am I missing something in the settings?

As a side note, the pen works exactly as it should (motion+click when touching screen) in bvnc 
[https://github.com/iiordanov/remote-desktop-clients/tree/master/bVNC](https://github.com/iiordanov/remote-desktop-clients/tree/master/bVNC)

Thanks for the interesting app!
Philippe

**Comment by AppGround-io** [comment]
You can activate the pen mode in mouse settings of the app to automatically active the left mouse button when touching down.

**Comment by phourglass** [comment]
Thanks for your response. I did some more tests.
The pen option was already activated.
But the touch to click option ("clics tactiles de la souris" in my local language)  was not.
After activating it, it works. 

I guess it should be automatically activated when the pen option is used ?
Anyhow. Problem solved 

Thanks
Philippe

**Reply by AppGround-io** [reply]
Thank you for investigating this. In the next version I will make some changes so that the pen mode does not depend on the touch click setting.
